parameter_files/FLAMINGO.yml

```diff
@@ -325,21 +315,30 @@ SOProperties:
     StarFormingGasMassFractionInMetals: true
     StellarCentreOfMass: true
     StellarCentreOfMassVelocity: true
-    StellarInertiaTensor: true
     StellarInitialMass: true
     StellarLuminosity: true
     StellarMass: true
     StellarMassFractionInIron: true
     StellarMassFractionInMetals: true
     StellarMassFractionInOxygen: true
     StellarMassInMetals: false
-    ReducedTotalInertiaTensor: true
-    ReducedGasInertiaTensor: true
-    ReducedDarkMatterInertiaTensor: true
-    ReducedStellarInertiaTensor: true
-    ReducedBaryonInertiaTensor: true
-    ThermalEnergyGas: true
     TotalInertiaTensor: true
+    GasInertiaTensor: true
+    DarkMatterInertiaTensor: true
+    StellarInertiaTensor: true
+    TotalInertiaTensorReduced: true
+    GasInertiaTensorReduced: true
+    DarkMatterInertiaTensorReduced: true
+    StellarInertiaTensorReduced: true
+    TotalInertiaTensorNoniterative: false
+    GasInertiaTensorNoniterative: false
+    DarkMatterInertiaTensorNoniterative: false
+    StellarInertiaTensorNoniterative: false
+    TotalInertiaTensorReducedNoniterative: false
+    GasInertiaTensorReducedNoniterative: false
+    DarkMatterInertiaTensorReducedNoniterative: false
+    StellarInertiaTensorReducedNoniterative: false
+    ThermalEnergyGas: true
```

Comment:
Is `ThermalEnergyGas` relevant to this PR?

Reply:
This just got moved when I was grouping all the tensor properties together, but it was set to true before anyway